webroot/results/person.php

```diff
@@ -66,6 +66,17 @@ function showBody () {
   if( $picture )
     echo "<center><img class='person' src='$picture' /></center>";
 
+  #--- Show the In Memoriam if any.
+  $inMemoriamArray = array(
+    "2008COUR01" => "https://www.worldcubeassociation.org/forum/viewtopic.php?f=9&t=2028",
+    "2003LARS01" => "https://www.worldcubeassociation.org/forum/viewtopic.php?f=9&t=1982",
+    "2012GALA02" => "https://www.worldcubeassociation.org/forum/viewtopic.php?f=9&t=1044",
+    "2008LIMR01" => "https://www.worldcubeassociation.org/forum/viewtopic.php?f=9&t=945",
+    "2008KIRC01" => "https://www.worldcubeassociation.org/forum/viewtopic.php?f=9&t=470",
+  );
+  if( array_key_exists($chosenPersonId, $inMemoriamArray) )
+    echo "<center><a target='_blank' href='".$inMemoriamArray[$chosenPersonId]."'>In Memoriam</a></center>";
```

Comment:
I wrote this first but vim highlighted it redredred:
![vim-unhappy](https://cloud.githubusercontent.com/assets/1007485/18567882/7ad7452e-7b9a-11e6-89df-bdb672fafa24.png)
so I did not even run it, but you're right it works!